Description: Disables the WebKit JIT on armel (fails to build)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/modules/web/src/main/native/Source/WTF/wtf/Platform.h
+++ b/modules/web/src/main/native/Source/WTF/wtf/Platform.h
@@ -738,6 +738,11 @@
 #define ENABLE_JIT 1
 #endif
 
+#if CPU(ARM) && WTF_ARM_ARCH_VERSION < 5
+#undef ENABLE_JIT
+#define ENABLE_JIT 0
+#endif
+
 /* The FTL *does not* work on 32-bit platforms. Disable it even if someone asked us to enable it. */
 #if USE(JSVALUE32_64)
 #undef ENABLE_FTL_JIT
